LinearDecayCorrelationFunction.h

```diff
@@ -0,0 +1,127 @@
+//#############################################################################
+//
+//    FILENAME:          LinearDecayCorrelationFunction.h
+//
+//    CLASSIFICATION:    Unclassified
+//
+//    DESCRIPTION:
+//
+//    Header for the piecewise linear decay correlation function class derived
+//    from the SPDCorrelationFunction base class.  This class is used to
+//    compute the correlation coefficient given a delta time.
+//    LIMITATIONS:       None
+//
+//
+//    SOFTWARE HISTORY:
+//     Date          Author   Comment
+//     -----------   ------   -------
+//     01-Dec-2021   JPK      Adapted from LinearDecayCorrelationModel
+//
+//    NOTES:
+//
+//#############################################################################
+
+#ifndef __CSM_LINEARDECAYCORRELATIONFUNCTION_H_
+#define __CSM_LINEARDECAYCORRELATIONFUNCTION_H_
+
+#include "SPDCorrelationFunction.h"
+#include "LinearDecayCorrelationModel.h"
+
+#include <vector>
+
+namespace csm
+{
+class CSM_EXPORT_API LinearDecayCorrelationFunction : public SPDCorrelationFunction
+{
+public:
+ 
+   LinearDecayCorrelationFunction();
+      //> This is the default contructor
+      //<
+      
+   LinearDecayCorrelationFunction(const std::vector<double>& initialCorrsPerSegment,
+                                  const std::vector<double>& timesPerSegment);
+   //> This constructor instantiates from the provided parameters.
+   //<
```

Comment:
Inconsistent indentation with other documentation comments in this header.  Ideally, you should indent these to 6 spaces.